http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/auth/WhitelistAuthProvider.scala

```diff
@@ -87,4 +87,14 @@ class WhitelistAuthProvider(config: Config, saProvider: ServiceAccountProvider[I
   )(implicit ev: ApplicativeAsk[IO, TraceId]): IO[Unit] = IO.unit
 
   override def serviceAccountProvider: ServiceAccountProvider[IO] = saProvider
+
+  /**
+   *
+   *
+   * @param internalId
+   * @return returns a list of Cluster Actions that the user has access to
+   */
+  override def getNotebookClusterActions(internalId: RuntimeInternalId, userInfo: UserInfo)(
+    implicit ev: ApplicativeAsk[IO, TraceId]
+  ): IO[List[String]] = IO.pure(List.empty)
```

Comment:
Instead of stubbing this out we should probably implement this the right way… something like
```
if (checkWhitelist(userInfo)) NotebookClusterAction.allActions else List.empty
```